{"uuid": "895649db-b7c8-40b3-9a8c-46e3dc5f7358", "children": ["4e4a3795-eb4a-4d78-afb2-dcfbeed5fdda", "fed5dda2-cebc-4d02-b8d7-7329cfb971f7", "79ad7dc3-d999-414b-86e5-b521706cce39"], "befores": [{"name": "setup", "status": "passed", "start": 1592458860973, "stop": 1592458898848}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: Unable to get browser\n\n", "trace": "  File \"c:\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\GOURAV\\PycharmProjects\\MultipleProjects\\TestCases\\conftest.py\", line 54, in setup\n    driver.close()\n  File \"c:\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 688, in close\n    self.execute(Command.CLOSE)\n  File \"c:\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1592458927062, "stop": 1592458927174}], "start": 1592458860973, "stop": 1592458927174}